source/wp-content/themes/wporg-developer-2023/functions.php

```diff
@@ -297,8 +299,8 @@ function pre_get_posts( $query ) {
 function register_nav_menus() {
```

Comment:
I think this whole function is dead now.

Reply:
Probably, but that's actually not related to this change:

- `devhub-cli-menu` is the nav in the blue banner on [the WP-CLI handbook](https://developer.wordpress.org/cli/commands/)
- `devhub-menu` is the nav in the blue banner everywhere else, but it's empty
- `reference-home-api` is the API column on [the reference page](https://developer.wordpress.org/reference/)

The 3 handbooks that don't use github do use nav menus on production for their ToC — I mentioned that in https://github.com/WordPress/wporg-developer/pull/189 but didn't implement that split behavior. 

<img width="464" alt="Screenshot 2023-02-03 at 1 17 26 PM" src="https://user-images.githubusercontent.com/541093/216677549-b493e10e-dd9a-49fc-a5e3-870ccb2c6f4a.png">

All that is to say, I'd rather not remove this as part of this PR, but we can remove it in general.